shell/client/accounts/account-settings.js

```diff
@@ -305,7 +304,7 @@ Template.sandstormAccountSettings.events({
 
   "click button.delete-account-real": function (ev, instance) {
     instance._deleteSubmitting.set(true);
-    Meteor.call("deleteOwnAccount", instance._deleteFeedback.get(), function (err) {
+    Meteor.call("deleteOwnAccount", instance.$(".feedback").val(), function (err) {
```

Comment:
Whoa I think this is the only use of jQuery anywhere in our code. Any reason you chose this instead of `.find(".feedback").value`?